utils/markdown.ts

```diff
@@ -16,6 +16,13 @@ export function resolveMarkdownRelativeLinks(
       if (path.startsWith("http") || path.startsWith("https")) {
         return match;
       }
+      // match a link (e.g. [./example](./example), will replace the link, not the text)
+      if (url) {
+        return match.replace(
+          `(${path})`,
+          `(${cdnBaseURL}/${path.replace(/^\.\//, "")})`,
```

Comment:
Can you please double-check if we could fix/improve the regex above? generating new `(${path})` seems hacky to replace while haven't tried it locally... Also what is a sample README URL we can test against?

Reply:
I've updated the regexp to simplify code and avoid too many if.